remote-api/src/main/java/org/alfresco/repo/webdav/WebDAVServlet.java

```diff
@@ -121,6 +121,13 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
             startTime = System.currentTimeMillis();
         }
 
+        if (request.getMethod().equals("POST") && !initParams.allowInsecurePOSTMethod())

```

Comment:
`WebDAV.METHOD_POST` could be used instead of `"POST"`

Reply:
done